js/app/models/veventModel.js

```diff
@@ -255,13 +359,22 @@ app.factory('VEvent', function(TimezoneService, FcEvent, SimpleEvent, ICalFactor
 							const rawDtstart = dtstartProp.getFirstValue('dtstart');
 							const rawDtend = context.calculateDTEnd(singleVEvent);
 
-							const dtstart = context.convertTz(rawDtstart, timezone.jCal);
-							const dtend = context.convertTz(rawDtend, timezone.jCal);
-							const fcEvent = FcEvent(iface, singleVEvent, dtstart, dtend);
+                            const dtstart = context.convertTz(rawDtstart, timezone.jCal);
+                            const dtend = context.convertTz(rawDtend, timezone.jCal);
+                            const fcEvent = FcEvent(iface, vevent, dtstart, dtend, {
```

Comment:
This is supposed to be `const fcEvent = FcEvent(iface, vevent, dtstart, dtend, {` instead of `const fcEvent = FcEvent(iface, singleVEvent, dtstart, dtend);`

Mistake while rebasing

Reply:
When this is fixed, Issue 2 of @a14a should be fixed